deploy/charts/venafi-kubernetes-agent/README.md

```diff
@@ -0,0 +1,162 @@
+```
+
+You can do this with the following command:
+
+```shell
+kubectl logs -n ${VENAFI_NAMESPACE} $(kubectl get pod -n ${VENAFI_NAMESPACE} -l app.kubernetes.io/instance=agent -o jsonpath='{.items[0].metadata.name}')
+```
+
+You can also check inb the Venafi Control Plane to see when the "Last Check In" was for your cluster.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| authentication | object | `{"secretKey":"privatekey.pem","secretName":"agent-credentials"}` | Authentication details for the Venafi Kuberente Agent |
+| authentication.secretKey | string | `"privatekey.pem"` | Key name in the references secret |
+| authentication.secretName | string | `"agent-credentials"` | Name of the secret containing the privatekey |
+| command | list | `[]` | Specify the command to run overriding default |
+| config | object | `{"clientId":"","configmap":{"key":null,"name":null},"period":"0h1m0s","server":"https://api.venafi.cloud/"}` | Configuration section for the Venafi Kubernetes Agent itself |
+| config.configmap | object | `{"key":null,"name":null}` | Sepcify ConfigMap details to load config from an existing resource This should be blankby default unless you have you own config |
+| config.period | string | `"0h1m0s"` | Send data back to the platform every minute unless changed |
+| config.server | string | `"https://api.venafi.cloud/"` | Overrides the server if using a proxy in your environment |
+| extraArgs | list | `[]` | Specify additional argument to pass to the agent |
+| fullnameOverride | string | `""` | Helm default setting, use this to shorten install name |
+| image.pullPolicy | string | `"IfNotPresent"` | Defaults to only pull if not already present |
+| image.repository | string | `"quay.io/jetstack/preflight"` | Default to Open Source image repository |
+| image.tag | string | `"v0.1.43"` | Overrides the image tag whose default is the chart appVersion |
+| imagePullSecrets | list | `[]` | Specify image pull credentials if using a prviate registry |
+| nameOverride | string | `""` | Helm default setting to override release name, leave blank |
+| nodeSelector | object | `{}` |  |
+| podAnnotations | object | `{}` |  |
+| podSecurityContext | object | `{}` |  |
+| replicaCount | int | `1` | default replicas, do not scale up |
+| resources.limits.cpu | string | `"500m"` |  |
+| resources.limits.memory | string | `"500Mi"` |  |
+| resources.requests.cpu | string | `"200m"` |  |
+| resources.requests.memory | string | `"200Mi"` |  |
+| securityContext.capabilities.drop[0] | string | `"ALL"` |  |
+| securityContext.readOnlyRootFilesystem | bool | `true` |  |
+| securityContext.runAsNonRoot | bool | `true` |  |
+| securityContext.runAsUser | int | `1000` |  |
+| serviceAccount.annotations | object | `{}` | Annotations to add to the service account |
+| serviceAccount.create | bool | `true` | Specifies whether a service account should be created @default true |
```

Comment:
why does it say @default true?

Reply:
Let me look into that one.

Reply:
It was because I had messed up the inserting of the default value. This default is inferred so I not longer needed it. basically, it should have been:

```markdown
# @default -- true
```

Removed as redundant